src/OidcClient.php

```diff
@@ -238,6 +238,33 @@ public function retrieveUserInfo(OidcTokens $tokens): OidcUserData
     return new OidcUserData($data);
   }
 
+  public function getIntrospection(OidcTokens $tokens): array
+  {
+    $headers = [];
+    if (in_array('client_secret_basic', $this->getTokenEndpointAuthMethods())) {
+      $headers = [
+        'Authorization: Basic ' . base64_encode(urlencode($this->clientId) . ':' . urlencode($this->clientSecret)),
+      ];
+    }
+
+    $jsonData = $this->urlFetcher->fetchUrl(
+      $this->getConfigurationValue('introspection_endpoint'),
```

Comment:
Please create a `getIntrospectionEndpoint` method to be consistent with the other endpoint retrievals.